arm/Microsoft.Subscription/aliases/deploy.bicep

```diff
@@ -0,0 +1,70 @@
+targetScope = 'managementGroup'
+
+@description('Required. EnrollmentAccount used for subscription billing')
+param enrollmentAccount string
+
+@description('Required. BillingAccount used for subscription billing')
+param billingAccount string
+
+@description('Required. Alias to assign to the subscription')
+param subscriptionAlias string
+
+@description('Required. Display name for the subscription')
+param subscriptionDisplayName string
+
+@allowed([
+  'Production'
+  'DevTest'
+])
+@description('Optional. The workload type of the subscription.')
+param subscriptionWorkload string = 'Production'
+
+@description('Optional. Owner Id of the subscription')
+param subscriptionOwnerId string = ''
+
+@description('Optional. Tenant Id of the subscription')
+param subscriptionTenantId string = ''
+
+@description('Optional. This parameter can be used to create alias for existing subscription Id')
+param subscriptionId string = ''
```

Comment:
As this module is about a subscription, would it make sense to make this more object oriented, not mentioning `subscription` in the parameters which specifically refer to the resource/object its managing?

Reply:
done